src/btreemap.rs

```diff
@@ -39,6 +39,9 @@ use std::borrow::Cow;
 use std::marker::PhantomData;
 use std::ops::{Bound, RangeBounds};
 
+#[cfg(test)]
+mod proptests;
```

Comment:
I assume this file is missing from the commit?

Reply:
Ouch, fixed.